### PR TITLE
Use `%%%`

### PR DESCRIPTION
**Pull request by armanbilge**
Believe it or not this is what caused Cats 2.9.0 to break Explore :)

### CODE REVIEW

**Review by cquiroz — APPROVED** [review]
Cool, thanks

File: lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala [hunk 1]
```diff
@@ -14,22 +15,12 @@ import java.io.FileInputStream
...
   def jsonResult(jsonFile: String): IO[Json] = {
-    val url  = getClass.getResource(jsonFile)
-    val file = Paths.get(url.toURI).toFile
-
-    inputStream(file).use { inStream =>
-      for {
-        str  <- IO.blocking(scala.io.Source.fromInputStream(inStream).mkString)
-        json <- IO.fromEither(parse(str))
-      } yield json
+    val path = Path(s"lucuma-schemas/src/test/resources") / jsonFile
```
Comment (rpiaggio) [inline lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala:20]: Is there a way to avoid the hardcoded path?

Comment (armanbilge) [reply lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala:20]: Scala.js doesn't know about resources. We can add the `BuildInfo` plugin and have the build generate sources with the path.

Comment (armanbilge) [reply lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala:20]: Or we can revert my changes here and just not run the tests on JS. They were not being run on JS before, maybe it's not that important.

Comment (rpiaggio) [reply lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala:20]: It's probably a good idea to run the tests in JS. Let's leave it as it is, having the hardcoded path is OK in this case.

**Pull request merged**